tests/pass/scalars.py

```diff
@@ -20,7 +20,6 @@ def __float__(self):
 np.int16(3.4)
 np.int32(4)
 np.int64(-1)
-np.uint8(A())
```

Comment:
Instead of removing, I would follow what is done here:

https://github.com/numpy/numpy-stubs/pull/70/files#diff-dd377d4c09025262e6f7008bcfe583dfR10

i.e.

```python
class B:
    def __int__(self):
        return 4

...

np.uint8(B())
```

(That way we keep a case checking that a class with `__int__` passes.)